lib/command/src/Obelisk/Command.hs

```diff
@@ -382,7 +384,7 @@ ob = \case
   ObCommand_Repl -> runRepl
   ObCommand_Watch -> inNixShell' $ static runWatch
   ObCommand_Shell so -> withProjectRoot "." $ \root ->
-    projectShell root False (_shellOpts_shell so) (_shellOpts_command so)
+    projectProc root False (_shellOpts_shell so) (pure . bash . BSU.fromString <$> _shellOpts_command so)
```

Comment:
Let's use `Just` instead of `pure` when we're not gaining anything from the additional polymorphism.

Reply:
The same could probably be said for `<$>` being just `map`. Using `<$>` for `map` is a *bit* more common so I'll let you decide that one. If it were me I'd use `map`...

Reply:
Lol of course now I realize the `pure` is for list and `<$>` is for `Maybe`... this is why extra polymorphism is a problem IMO: It makes reading code harder.